test/infamy/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .env import Env
 from .netns import IsolatedMacVlan
 from .sniffer import Sniffer
+from .ssdp import SsdpClient
```

Comment:
This seems specific enough that we might leave it out of the default case.

I.e. the tests that need it can `from infamy.ssdp import SsdpClient` explicitly.

Reply:
Good point, didn't really know what `__init___.py` was for, will drop in a follow-up commit.

Reply:
You mean you'll add it to this PR, or to a later one?

Reply:
To this PR, sorry for betting so slow.

Reply:
Yeah, the famously unproductive @troglobit! :wink: 

Take all the time you need, I just didn't want to leave this PR in limbo :+1: